modules/jumpstart_ui/templates/layouts/one-column.html.twig

```diff
@@ -20,7 +20,6 @@
 {% if settings.columns|render is not empty and settings.columns != "default" %}
   {% if region_attributes.main is iterable %}
     {% set main_attributes = main_attributes.removeClass('flex-12-of-12').addClass(settings.columns) %}
-    {% set attributes = attributes.addClass('centered-content') %}
```

Comment:
Why would we remove this?

Reply:
But then the h1 is pushed against the left side of the screen, making it worse than before.
I also feel like this could have a more negative impact than we are aware. This template is used in many places outside the basic page. I would want a full 100% review of every place and every scenario that this touches. It's too risky IMO.

Reply:
You are correct. I answered too quickly earlier. I'll work on it on Monday. I feel awful this afternoon.

Reply:
I fixed it.  Will you take a look on Monday?